src/corebluetooth.cs

```diff
@@ -773,14 +783,17 @@ interface CBPeripheralManager {
 		[Protocolize]
 		CBPeripheralManagerDelegate Delegate { get; set; }
 
-		[Export ("delegate", ArgumentSemantic.Assign)]
+		[Export ("delegate", ArgumentSemantic.Weak)]
 		NSObject WeakDelegate { get; set; }
 
 		[Export ("isAdvertising")]
 		bool Advertising { get; }
 
+#if XAMCORE_4_0
+		// Removed in iOS 10 – Breaking change.
```

Comment:
Not a breaking change. The selector now exists in the base type and the enum is compatible. Still it's the right choice to remove it but

* the comment should be updated;

* the existing API should be moved (from here) to the manual bindings (under `!XAMCORE_4_0`) and call the base property (casting the enum);